master/getting-started/mesos/installation/docker.md

```diff
@@ -5,65 +5,92 @@ title: Installing Calico for the Docker Containerizer
+#### 1. etcd
 
-```shell
-    --cluster-store=etcd://<ETCD HOST>:<PORT>
+Calico uses etcd as its datastore. Ensure you have an instance of etcd running,
+and that it is accessible from all Agents in your cluster.
+
+To maximize availability, we recommend running an etcd cluster across your Mesos masters.
+See [etcd's clustering guide](https://coreos.com/os/docs/latest/cluster-architectures.html)
+for more information.
+
+For testing, you can run a single instance of etcd using docker on your Master
+with the following command, ensuring you've correctly set `$ETCD_IP`:
+```
+docker run --detach \
+	--net=host \
+	--name etcd quay.io/coreos/etcd:v2.0.11 \
+	--advertise-client-urls "http://$ETCD_IP:2379" \
+	--listen-client-urls "http://$ETCD_IP:2379,http://127.0.0.1:2379"
 ```
 
-Replacing `<ETCD HOST>:<PORT>` with the appropriate `hostname:port`
-for your etcd cluster.
+#### 2. Docker Configured with Cluster Store
 
-## Install and Run Calico
-It is very easy to install Calico to use with the
-Docker Containerizer.
+Under the covers, Calico networks Docker tasks for Mesos with its Docker CNM plugin.
+Multihost Networking in Docker requires that
+each Agent's Docker daemon must be configured with a cluster store.
 
-1. On each Mesos Agents, download the `calicoctl` command-line tool:
+Though Docker's configured cluster-store does not have to be the same as Calico's,
+for simplicity, users can configure Docker to use the same by setting the
+following flag in `/etc/docker/daemon.json`:
 
-```shell
-curl -o /usr/bin/calicoctl -L http://www.projectcalico.org/builds/calicoctl
-chmod a+x calicoctl
+```json
+{
+  "cluster-store":"etcd://<ETCD_IP>:<ETCD_PORT>"
+}
 ```
 
-2. Launch the `calico/node` container.
+Replace `<ETCD_IP>` and `<ETCD_PORT>` with the appropriate `hostname:port` for your etcd cluster.
+
+#### 3. Docker Containerizer Enabled for Mesos Agents
 
-For production deployments, we recommend running the
-container as a service. Visit our guide on [running Calico
-as a service]({{site.baseurl}}/{{page.version}}/usage/configuration/as-service) to learn how to do this.
+By default, Mesos enables only the "Mesos" Containerizer. Ensure
+the Docker Containerizer is also enabled on each Agent.
 
-For test environments that you would like to get up and running
-quickly, you can launch the container with `calicoctl`:
+If you are using the default `mesos-init-wrapper` from the official Mesos package,
+you can enable the Docker Containerizer with the following command:
 
 ```shell
-sudo ETCD_ENDPOINTS=http://<ETCD HOST:PORT> ./calicoctl noderun
+$ sh -c 'echo docker > /etc/mesos-slave/containerizers'
+$ systemctl restart mesos-slave.service
 ```
 
-Again, be sure to set the ETCD_ENDPOINTS to the correct value for your etcd cluster.
 
-3. Ensure calico's services are running by checking for the Calico container in Docker:
+## Installing Calico
+
+To install Calico, on each Mesos Agent, download the `calicoctl` command-line tool:
```

Comment:
I feel that the we either should:
-  Move the warning about this being for trial only being moved to the top
-  or, just say that you need to add a systemd or similar and be done with it.

I'm worried that we'll say run `calicoctl node run` and somewhere down the line someone will think that it's deployed and resilient.   WDYT?   

I'd probably just move the warning to the top.  I have more ideas about what i want the install to look like in the future.